src/main/resources/twitter-demo/static/js/newTweetBook.js

```diff
@@ -0,0 +1,655 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+$(function () {
+
+  // Add a format function to String
+  if (!String.prototype.format) {
+    String.prototype.format = function () {
+      var args = arguments;
+      return this.replace(/{(\d+)}/g, function (match, number) {
+        return typeof args[number] != 'undefined'
+          ? args[number]
+          : match
+          ;
+      });
+    };
+  }
+
+
+  // Initialize connection to AsterixDB. Just one connection is needed and contains
+  // logic for connecting to each API endpoint. This object A is reused throughout the
+  // code but does not store information about any individual API call.
+  A = new AsterixDBConnection({
+
+    // We will be using the geo dataverse, which we can configure either like this
+    // or by following our AsterixDBConnection with a dataverse call, like so:
+    // A = new AsterixDBConnection().dataverse("geo");
+    "dataverse": "twitter",
+
+    // Due to the setup of this demo using the Bottle server, it is necessary to change the
+    // default endpoint of API calls. The proxy server handles the call to http://localhost:19002
+    // for us, and we reconfigure this connection to connect to the proxy server.
+    "endpoint_root": "/",
+
+    // Finally, we want to make our error function nicer so that we show errors with a call to the
+    // reportUserMessage function. Update the "error" property to do that.
+    "error": function (data) {
+      // For an error, data will look like this:
+      // {
+      //     "error-code" : [error-number, error-text]
+      //     "stacktrace" : ...stack trace...
+      //     "summary"    : ...summary of error...
+      // }
+      // We will report this as an Asterix REST API Error, an error code, and a reason message.
+      // Note the method signature: reportUserMessage(message, isPositiveMessage, target). We will provide
+      // an error message to display, a positivity value (false in this case, errors are bad), and a
+      // target html element in which to report the message.
+      var showErrorMessage = "Asterix Error #" + data["error-code"][0] + ": " + data["error-code"][1];
+      var isPositive = false;
+      var showReportAt = "report-message";
+
+      reportUserMessage(showErrorMessage, isPositive, showReportAt);
+    }
+  });
+
+  // UI-Elements: create map and add controls
+  map = L.map('map').setView([39.5,-96.35,], 4);
+  L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
+      attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
+      maxZoom: 18,
+      id: 'jeremyli.p6f712pj',
+      accessToken: 'pk.eyJ1IjoiamVyZW15bGkiLCJhIjoiY2lrZ2U4MWI4MDA4bHVjajc1am1weTM2aSJ9.JHiBmawEKGsn3jiRK_d0Gw'
```

Comment:
Seems that public token is safe.
https://www.mapbox.com/help/define-access-token/